internal/pkg/auth/sso/auth_state.go

```diff
@@ -30,20 +30,28 @@ var (
 			ssoProviderIdentifier: "https://confluent-dev.auth0.com/api/v2/",
 			ssoProviderScope:      "email%20openid%20offline_access",
 		},
-		"fedramp-internal": {
-			ssoProviderDomain: "confluent-infra-us-gov.oktapreview.com/oauth2/v1",
+		"devel-us-gov": {
+			ssoProviderDomain: "confluent-devel-us-gov.oktapreview.com/oauth2/v1",
 			ssoProviderScope:  "openid+profile+email+offline_access",
 		},
-		"stag": {
-			ssoProviderDomain:     "login-stag.confluent-dev.io/oauth",
-			ssoProviderIdentifier: "https://confluent-stag.auth0.com/api/v2/",
-			ssoProviderScope:      "email%20openid%20offline_access",
+		"infra-us-gov": {
+			ssoProviderDomain: "confluent-infra-us-gov.oktapreview.com/oauth2/v1",
+			ssoProviderScope:  "openid+profile+email+offline_access",
 		},
 		"prod": {
 			ssoProviderDomain:     "login.confluent.io/oauth",
 			ssoProviderIdentifier: "https://confluent.auth0.com/api/v2/",
 			ssoProviderScope:      "email%20openid%20offline_access",
 		},
+		"prod-us-gov": {
+			ssoProviderDomain: "confluent-prod-us-gov.oktapreview.com/oauth2/v1",
```

Comment:
@xuhui-lu is this correct? I'm not sure we're using `oktapreview` for prod.

Reply:
I believe for prod-us-gov it is confluent-prod-us-gov.okta.com, not oktapreview

Reply:
```suggestion
			ssoProviderDomain: "confluent-prod-us-gov.okta.com/oauth2/v1",
```